test/ResourcesClassLibraryNoAttribute/project.json

```diff
@@ -0,0 +1,17 @@
+﻿{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "warningsAsErrors": true,
+    "keyFile": "../../tools/Key.snk"
+  },
+  "dependencies": {
+    "NETStandard.Library": "1.6.0"
```

Comment:
:ok: guess I was mostly curious why the project needed this particular .NET standard.